### PR TITLE
Add new 16.04 ubuntu image version for s390x

### PR DESCRIPTION
**Pull request by liwbj**
Add new 16.04 ubuntu image version for s390x

### CODE REVIEW

**Comment by sathnaga** [comment]
Acked

**Comment by liwbj** [comment]
@lmr can you take a look for this?

File: shared/cfg/guest-os/Linux/Ubuntu/16.04-server.s390x.cfg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+- 16.04-server.s390x:
+    image_name += -16.04-server-s390x
+    vm_arch_name = s390x
+    os_variant = ubuntutrusty
+    unattended_install, svirt_install:
+        kernel = images/ubuntu-server-16-04-s390/vmlinuz
+        initrd = images/ubuntu-server-16-04-s390/initrd.gz
+        boot_path = install
+    unattended_install.cdrom, svirt_install:
+        unattended_file = unattended/Ubuntu-16-04.preseed
```
Comment (xutian) [inline shared/cfg/guest-os/Linux/Ubuntu/16.04-server.s390x.cfg]: Not found unattended/Ubuntu-16-04.preseed file,  it looks you missed this file in your patch.

Thanks,
Xu

**Comment by xutian** [comment]
@liwbj  Can you update this PR when free ?

Thanks,
Xu

**Comment by lmr** [comment]
Ah, looks good

**Comment by lmr** [comment]
Merged, thanks!

**Pull request merged**